pkg/utils/build/quay.go

```diff
@@ -0,0 +1,60 @@
+package build
+
+import (
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	. "github.com/redhat-appstudio/image-controller/pkg/quay"
```

Comment:
please don't use dot imports for other packages than ginkgo and gomega

Reply:
Thanks. Updated now.